Stackup-Rwanda/knights-bn-frontend
# #172499349 Add multi language support

#### What does this PR do?
Enables a user to switch between the languages for a better experience.

#### Description of Task to be completed?
- Auto detect the preferred language of the user based on the browser settings.
- Add support for English language.
- Add support for Ikinyarwanda language.
- Add support for French language.

#### How should this be manually tested?
- run `git clone https://github.com/Stackup-Rwanda/knights-bn-frontend.git` to have the repo locally.
- run `cd /knights-bn-frontend.git` to get inside the project directory.
- run `git checkout -t origin/ft-multi-language-support-172499349` to switch to this branch.
- run `npm i` to install all dependencies.
- run `npm dev-start` to start the server.
- the browser should start by itself to the homePage.

#### What are the relevant pivotal tracker stories?
[link](https://www.pivotaltracker.com/story/show/172499349)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/30593186/82002261-a5711d00-965d-11ea-8d55-49c7eb4b37ff.png)

## English version
![image](https://user-images.githubusercontent.com/30593186/82002314-c9346300-965d-11ea-832c-8083ace29f41.png)


## Ikinyarwanda version
![image](https://user-images.githubusercontent.com/30593186/82002302-be79ce00-965d-11ea-874d-6cd77213ed83.png)

## French version
![image](https://user-images.githubusercontent.com/30593186/82002376-e9642200-965d-11ea-823c-f0366c9f7d0a.png)


#### Questions:
